rundeckapp/grails-app/views/scm/performAction.gsp

```diff
@@ -255,7 +255,7 @@
                                         <div class="checkbox flex-none"  style="margin-top: 0;margin-left: 20px;">
                                                 <g:checkBox name="chosenTrackedItem"
                                                             id="chosenTrackedItem${trackedItem.id}"
-                                                            value="${trackedItem.id}"
+                                                            value="${trackedItem.id?.replaceAll("\"", '&quot;')}"
```

Comment:
I think we should find a more generic way to encode this as a html-safe string.
This would fix it for quotes, but what if there's another html artifact? It could even lead to a vulnerability.